src/Fetchers/TMSFetcher/TMSFetcher.ts

```diff
@@ -75,41 +75,50 @@ export class TMSFetcher extends Fetcher {
      * Fetch the matches from TMS.
      */
     protected async fetch() {
-        this.log("info", "Fetching competitions.");
-        const competitions = await this.fetchCompetitions();
-        const promises = [];
-
-        this.log("info", `Found ${competitions.size} competitions.`);
-        this.log("info", "Fetching matches and creating competition files.");
-
-        for (const competition of competitions.values()) {
-            // Fetch match for every competition
-            const matchPromise = this.fetchMatches(competition);
-            matchPromise.then(result => {
-                competition.getMatches().push(...result.values());
-                return ICSCreator.createCompetitionICS(competition);
-            });
-
-            promises.push(matchPromise);
+        try {
```

Comment:
Why is the try-catch necessary here? Which errors are we possibly trying to catch? I'd like a better solution for this than simply catching all errors and logging them, because this will not fail the auto-update script if something goes wrong.

Reply:
Relic from testing, my bad.